examples/cells_quota.ts

```diff
@@ -28,10 +28,11 @@
  *   limitations under the License.                                           *
  *                                                                            *
 \* -------------------------------------------------------------------------- */
-import * as helpers from "../auraescript/gen/helpers.ts";
+import * as aurae from "../auraescript/gen/aurae.ts";
 import * as runtime from "../auraescript/gen/cells.ts";
```

Comment:
while you're here, can this be `import * as cells`?

Reply:
I agree this would look better, but as @future-highway said (https://github.com/aurae-runtime/aurae/pull/427#issuecomment-1445377988) changing the import will break IDE support
I didn't find a way to hint typescript LSP for globally existing functions or import paths, I haven't looked too hard though

Reply:
I think dma was suggesting `import * as cells from "../auraescript/gen/cells.ts"`.

What is called cells.ts used to be runtime.ts and we missed a couple of renames.

Reply:
Oh! I get it now